segmentation_models_pytorch/fpn/decoder.py

```diff
@@ -100,7 +103,10 @@ def forward(self, x):
         s3 = self.s3(p3)
         s2 = self.s2(p2)
 
-        x = s5 + s4 + s3 + s2
+        if self.merge_policy == 'add':
+            x = s5 + s4 + s3 + s2
+        elif self.merge_policy == 'cat':
+            x = torch.cat([s5, s4, s3, s2])
```

Comment:
In case of 'cat' `final_conv` should have segmentation_channels * 4 input channels (line 89)
And set `dim=1` for cat (deafult is 0, which is wrong).

Reply:
Fixed, thanks.